### PR TITLE
Initial script

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <dlorenc@google.com>

### CODE REVIEW

File: scripts/step-0.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+if [ -z "$GITHUB_USER" ]; then
+    echo "Set GITHUB_USER"
+    exit
+fi
+
+# Dump the git state
+git status
+git remote -v
+
+# Setup forks
```
Comment (asraa) [inline scripts/step-0.sh:15]: should we log/warn the user they need to fork from the UI?

File: scripts/step-0.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+git remote rm origin
+git remote add origin git@github.com:"$GITHUB_USER"/root-signing.git
+git remote -v
+
+git clean -d -f
+git checkout main
+git pull upstream main
+git rev-parse HEAD
+
+# build the tuf binary and print the digest
```
Comment (asraa) [inline scripts/step-0.sh]: print the digest TODO?

File: scripts/step-1.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+git remote -v
+
+# Add the key!
+./tuf add-key -repository $REPO
+git status
+git checkout -b add-key
+git commit -s -a -m "Adding initial key for ${GITHUB_USER}"
+git push -f origin add-key
+
+# Open the browser
```
Comment (asraa) [inline scripts/step-1.sh:26]: nice!!!

File: scripts/step-1.5.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+git status
+git remote -v
+
+git clean -d -f
+git checkout main
+git pull upstream main
+git status
+
+# Setup the root
+# TODO
```
Comment (asraa) [inline scripts/step-1.5.sh]: the non-git stuff will look something like
`./tuf init -repository $REPO -target targets/fulcio.crt.pem [-target <OTHER STUFF>]`

if it's easier i could make this point to a target directory

File: scripts/step-5.sh [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+git status
+git remote -v
+
+git clean -d -f
+git checkout main
+git pull upstream main
+git status
+
+# Sign the root and targets
+# TODO, publish
```
Comment (asraa) [inline scripts/step-5.sh]: ./tuf publish -repository $REPO

**Review by asraa — COMMENTED** [review]
Yay!!! Thanks

File: scripts/step-0.sh (hunk 1 above)
Comment (dlorenc) [reply scripts/step-0.sh:15]: I think we just have to tell them that manually...

File: scripts/step-0.sh (hunk 2 above)
Comment (dlorenc) [reply scripts/step-0.sh]: Dropped it, it's too confusing with different platforms.

File: scripts/step-1.5.sh (hunk 4 above)
Comment (dlorenc) [reply scripts/step-1.5.sh]: DId i get this right?

Comment (asraa) [reply scripts/step-1.5.sh]: yep!

File: scripts/step-1.5.sh [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+git status
+git remote -v
+
+git clean -d -f
+git checkout main
+git pull upstream main
+git status
+
+# Setup the root
+./tuf init -repository $REPO -target targets/fulcio.crt.pem
```
Comment (asraa) [inline scripts/step-1.5.sh:25]: git checkout branch here?

**Review by asraa — APPROVED** [review]
LGTM!

**Pull request merged**